typing_extensions/src_py2/test_typing_extensions.py

```diff
@@ -616,6 +616,13 @@ class Proto(Protocol):
             class Concrete(Proto):
                 pass
 
+    def test_none_treated_correctly(self):
+        @runtime
+        class P(Protocol):
+            x = None  # type: int
+        class B(object): pass
```

Comment:
Do you test the variants where `x` is defined for the instance of `B` (`None` / not `None`) or where `x` has value `None` / not `None` in the body of `B`?

Reply:
I think I test only few variants/combinations. I will add more tests.